### PR TITLE
Add unknown recipient button when composing new message

### PR DESCRIPTION
**Pull request by LunarX**
Depends on #599

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/ui/main/newMessage/RecipientFieldView.kt [hunk 1]
```diff
@@ -105,7 +107,9 @@ class RecipientFieldView @JvmOverloads constructor(
                 onAddUnrecognizedContact = {
                     val input = autoCompleteInput.text.toString()
                     if (input.isEmail()) addRecipient(email = input, name = input)
+                    else setSnackBar(R.string.addUnknownRecipientInvalidEmail)
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/main/newMessage/RecipientFieldView.kt:110]: Add braces.

File: app/src/main/java/com/infomaniak/mail/ui/main/newMessage/RecipientFieldView.kt [hunk 2]
```diff
@@ -105,7 +107,9 @@ class RecipientFieldView @JvmOverloads constructor(
...
                 },
+                setSnackBar = { setSnackBar(it) }
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/main/newMessage/RecipientFieldView.kt:112]:
```suggestion
                setSnackBar = { setSnackBar(it) },
```

File: app/src/main/res/drawable/ic_unknown_user_avatar.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (KevinBoulongne) [inline app/src/main/res/drawable/ic_unknown_user_avatar.xml:1]: Add header.

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -191,7 +195,7 @@
     <string name="onBoardingDescription2">Définissez les actions que vous effectuez le plus souvent au balayage vers la droite et la gauche, et gagnez un temps précieux</string>
     <string name="onBoardingDescription3">Effectuez des actions sur de multiples messages simultanément</string>
     <string name="onBoardingDescription4">Gérez tous vos e-mails Infomaniak depuis l’app !</string>
-    <string name="onBoardingTitle1">Choisissez entre 2 thèmes,\nrose ou bleu ?</string>
+    <string name="onBoardingTitle1">Choisissez une couleur d\'accentuation,\nrose ou bleu ?</string>
```
Comment (KevinBoulongne) [inline app/src/main/res/values-fr/strings.xml:198]:
```suggestion
    <string name="onBoardingTitle1">Choisissez une couleur d’accentuation,\nrose ou bleu ?</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -42,6 +42,9 @@
     <string name="actionSpam">Signaler comme spam</string>
     <string name="actionStar">Ajouter aux favoris</string>
     <string name="actionUnstar">Retirer des favoris</string>
+    <string name="addUnknownRecipientAlreadyUsed">L\'adresse mail est déjà utilisée</string>
```
Comment (KevinBoulongne) [inline app/src/main/res/values-fr/strings.xml:45]: Use `’`.

Modify it on Loco too.

File: app/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -42,6 +42,9 @@
...
+    <string name="addUnknownRecipientInvalidEmail">L\'adresse mail n\'est pas valide</string>
```
Comment (KevinBoulongne) [inline app/src/main/res/values-fr/strings.xml:46]: Use `’`.

Modify it on Loco too.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**